packages/playground-editor/src/useDarkMode.ts

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+
+const useMediaQuery = (query: string) => {
```

Comment:
Any use for exporting/extracting this out for other usage down the road?

Reply:
My thinking on this: keep things concealed until they need to be revealed. If we have a use-case for this down the road, we can slap an `export` on the front and use it.